src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -312,9 +345,13 @@ protected function prepareForDenormalization($data)
      * @param array|bool $allowedAttributes
      *
      * @return \ReflectionMethod|null
+     *
+     * @deprecated since Symfony 5.3, use "instantiator_constructor" field in context array instead.
      */
     protected function getConstructor(array &$data, string $class, array &$context, \ReflectionClass $reflectionClass, $allowedAttributes)
     {
+        trigger_deprecation('symfony/serializer', '5.3', 'The "%s()" method is deprecated. Use "%s" field in context array instead.', __METHOD__, Instantiator::INSTANTIATOR_CONSTRUCTOR);
```

Comment:
You can do something like this to trigger this deprecation only if this method is overrode: https://3v4l.org/42Qro